crates/bevy_ecs/src/entity/map_entities.rs

```diff
@@ -39,7 +43,33 @@ pub trait MapEntities {
     ///
     /// Implementors should look up any and all [`Entity`] values stored within and
     /// update them to the mapped values via `entity_mapper`.
-    fn map_entities(&mut self, entity_mapper: &mut EntityMapper);
+    fn map_or_gen_entities(&mut self, entity_mapper: &mut EntityMapper);
+
+    /// Updates all [`Entity`] references stored inside using `entity_mapper`.
+    ///
+    /// Only updates the references for which there is a mapping.
+    fn map_entities(&mut self, entity_mapper: &mut SimpleEntityMapper);
```

Comment:
This can take `&SimpleEntityMapper` unless you plan to add new methods to `SimpleEntityMapper` that require `&mut self`

Reply:
correct